en/orm/query-builder.rst

```diff
@@ -745,7 +754,7 @@ Which will generate the following SQL looking like::
 It is also possible to build expressions using SQL functions::
 
     $query = $articles->find()
-        ->where(function ($exp, $q) {
+        ->where(function (QueryExpression , Query $q) {
```

Comment:
what happened to the $exp?

Reply:
See 2b091415e1f73001d1a34dda971180e459cdc8d4